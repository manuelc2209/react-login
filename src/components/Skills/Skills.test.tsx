import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { Skills } from '..';

it('renders snapshot correctly', () => {
    const tree = renderer.create(<Skills />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Skills />, div);
    ReactDOM.unmountComponentAtNode(div);
});
