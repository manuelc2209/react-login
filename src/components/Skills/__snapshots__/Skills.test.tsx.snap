// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<div
  className="sc-jJoQJp lfquEV"
>
  <span
    className="sc-ikJyIC hFggZb"
  >
    My skills:
  </span>
</div>
`;
